docs/production/giphy-gif-integration.md

```diff
@@ -41,7 +41,9 @@ Zulip server.  Your users can now use the integration as described in
 [the help center article][help-center-giphy].  (A browser reload may
 be required).
 
-
+You can also set the maximum allowed rating of GIFs that is
+retrieved from GIPHY in your organization settings or disable
+GIPHY using it.
```

Comment:
I think we can leave change sentence off -- it's explained in the help center article and I want that to be the authoritative reference of what configuration features we have.